<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>954</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>778</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1224</width>
    <height>954</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Post-pancreaticoduodenectomy hemorrhage detector (p-PHD)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/title.png</normaloff>img/title.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 148, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>690</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 87 9pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Prediction accuracy:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Accuracy">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>690</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Age">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Albumin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LiverDisease">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>630</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VascularResection">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="INR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>510</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Height">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>210</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Albumin (g/dl)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>630</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 87 9pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Is it possible to experience PPH:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HCT">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OperatingTime">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Age (year)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #F3ECDB;
font: 75 18pt &quot;Times New Roman&quot;;
color: #C8515C;</string>
    </property>
    <property name="text">
     <string>Post-pancreaticoduodenectomy hemorrhage detector (p-PHD)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>310</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Operating time (min)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>INR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>410</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Height (cm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>BUN (mg/dl)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>260</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>HCT (vol%)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BUN">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Cr (mg/dl)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>560</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>ASA class</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>360</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Vascular resection (yes=1/no=0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Cr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ASAclass">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Click_For_Result">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>630</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 238, 225);
color: rgb(0, 0, 0);
font:8pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Click_For_Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 87 9pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>preoperative and intraoperative indices</string>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>735</y>
      <width>831</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Risk">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>630</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:8pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Risk</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
